arch/eventloop_zephyr/eventloop_zephyr.h

```diff
@@ -0,0 +1,243 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2017 (c) Stefan Profanter, fortiss GmbH
+ *    Copyright 2021 (c) Christian von Arnim, ISW University of Stuttgart (for VDW and
+ * umati) Copyright 2021 (c) Fraunhofer IOSB (Author: Julius Pfrommer) Copyright 2021 (c)
+ * Fraunhofer IOSB (Author: Jan Hermes)
+ */
+
+#ifndef UA_EVENTLOOP_Zephyr_H_
+#define UA_EVENTLOOP_Zephyr_H_
+
+#include <open62541/config.h>
+#include <open62541/plugin/eventloop.h>
+
+#if defined(UA_ARCHITECTURE_ZEPHYR)
+
+#include <errno.h>
+
```

Comment:
This file contains a lot of internals.
I would not include it from the config_default plugin.
Rather include the header in /plugins/include/... that only defines the constructor function for the eventloop and the tcp layer.

Reply:
The `eventloop_zephyr.h` defines the macro `UA_LOG_SOCKET_ERRNO_WRAP`, which relies on `errno`. In my opinion, the best approach is to either also remove this macro or stick with the current solution. Any other approach would leave the instructions unclear.